plugins/release-manager/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@backstage/plugin-release-manager",
+  "version": "0.4.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "clean": "backstage-cli clean",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "start": "backstage-cli plugin:serve"
+  },
+  "dependencies": {
+    "@backstage/core": "^0.4.0",
+    "@backstage/theme": "^0.2.2",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.45",
+    "@material-ui/styles": "^4.10.0",
+    "axios": "^0.19.2",
+    "axios-hooks": "^2.1.0",
+    "lodash": "^4.17.19",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-use": "^15.3.3"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.4.1",
+    "@backstage/dev-utils": "^0.1.6",
+    "@backstage/test-utils": "^0.1.5",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^10.4.1",
+    "@testing-library/user-event": "^12.0.7",
+    "@types/jest": "^26.0.7",
+    "@types/node": "^12.0.0",
+    "@types/lru-cache": "^5.1.0",
+    "jest-fetch-mock": "^3.0.3"
```

Comment:
We (ben) worked hard to remove this, use `msw` instead 😅 

See https://backstage.io/docs/architecture-decisions/adrs-adr007